backend/webui_service/webui_init.go

```diff
@@ -96,6 +98,9 @@ func (a *WebuiApp) Start(tlsKeyLogPath string) {
 	self := webui_context.GetSelf()
 	self.UpdateNfProfiles()
 
+	wg := sync.WaitGroup{}
+	self.BillingServer = billing.OpenServer(&wg)
+
```

Comment:
The waitGroup is used to ensure the main goroutine will waiting for the other goroutines exit gracefully when application terminates.
Should we append the graceful shutdown processing here (wg.Add()  and wg.Wait())?